system/handlers/admin/DataManager.cfc

```diff
@@ -2423,6 +2423,16 @@ component extends="preside.system.base.AdminHandler" {
 
 		validationResult = validateForm( formName=arguments.formName, formData=formData, stripPermissionedFields=arguments.stripPermissionedFields, permissionContext=arguments.permissionContext, permissionContextKeys=arguments.permissionContextKeys );
 
+		args.formData         = formData;
```

Comment:
`args` has not been initialized here. ie, we should probably have it in the arguments list ( `struct args = {}`)

Reply:
I'm initialized via `var args = arguments;` by following how the DatamMnager._addRecordAction instead. Might also useful to have more params passing to customization point